seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-clickhouse/src/main/java/org/apache/seatunnel/flink/clickhouse/sink/client/ClickhouseClient.java

```diff
@@ -212,4 +221,21 @@ public ClickhouseTable getClickhouseTable(String database, String table) {
 
     }
 
+    /**
+     * Localization the engine in clickhouse local table's createTableDDL to support specific engine.
+     * For example: change ReplicatedMergeTree to MergeTree.
+     *
+     * @param engine original engine of clickhouse local table
+     * @param ddl createTableDDL of clickhouse local table
+     * @return createTableDDL of clickhouse local table which can support specific engine
+     * TODO: support more engine
+     */
+    public static String localizationEngine(String engine, String ddl) {
```

Comment:
Please remove the static, this is not a good practice, if you only want to call this method in UT.

Reply:
I have updated, please have a look.

Reply:
OK, I see your updated code, you remove the static, it's OK to me. In fact, I perfer to change the public to default, this is not a problem.